cfme/common/provider.py

```diff
@@ -66,36 +66,43 @@ class Credential(cfme.Credential, Updateable):
         """
         @property
         def form(self):
-            fields = [
-                ('token_secret_55', Input('bearer_token')),
-                ('google_service_account', Input('service_account')),
-            ]
-            tab_fields = {
-                "Default": [
+            if self.domain == 'azure' and version.current_version() >= '5.6.1':
```

Comment:
This is what I came up with to fix the unique azure implementation.  I couldn't find provider.type or its equivelant and didn't want to add way to much code to find it.